src/kernel/chainparams.cpp

```diff
@@ -132,7 +132,7 @@ class CMainParams : public CChainParams {
         // release ASAP to avoid it where possible.
         vSeeds.emplace_back("seed.bitcoin.sipa.be."); // Pieter Wuille, only supports x1, x5, x9, and xd
         vSeeds.emplace_back("dnsseed.bluematt.me."); // Matt Corallo, only supports x9
-        vSeeds.emplace_back("dnsseed.bitcoin.dashjr.org."); // Luke Dashjr
+        vSeeds.emplace_back("dnsseed.bitcoin.dashjr-list-of-p2p-nodes-maybe-malware.us."); // Luke Dashjr
```

Comment:
I am not sure why you got a domain with 'malware' in its name from namecheap for using as DNS seed in bitcoin core.

Related question (although out of scope for this PR): If there are so many issues with DNS seeds, why not disable `dnsseed` and `fixedseeds`? Instead 9 developers running DNS seeds can provide a node address which can be used for `seednode` to get node addresses using GETADDR. Or make bootstrapping work like tor nodes for all nodes.

Reply:
There are advantages and disadvantages to both DNS seeds and fixed seeds. Having both is good.

Discussing this doesn't need to happen here on this straightforward pull-req.